app/views/participants/fields/_financial.html.erb

```diff
@@ -5,7 +5,7 @@
 		<%= f.input :free_reduced_lunch_signed_up, :label => "Signed up" %>
 	<% end %>
 
-	<%= f.input :household_size, :collection => 1..20, :as => :select, :label => "Number of persons in household", :hint => "Include the student in this count." %>
+	<%= f.input :household_size, :collection => 1..20, :as => :select, :label => "Number of persons in household"%>
```

Comment:
Yes, we should populate these into `seeds.rb` so they don't get lost for new tenants. The problem with `rake db:seed`, though, is that it doesn't check for duplicates so we would end up duplicating all the current seeds. So I would propose that you add the values to `seeds.rb` but *do not run* the rake task. Instead, for the other tenants, we'll just run it manually to get them up to date, and then any new tenants will get the new values through the regular seeding process.

Make sense to you both?